app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use App\Blog;
+use Illuminate\Support\Facades\Auth;
+use Carbon\Carbon;
```

Comment:
You also did not use Carbon yet, so no need to put here.

Reply:
Fixes issue(Delete)